README.md

```diff
@@ -57,32 +57,29 @@ Optionally [VSCode](https://code.visualstudio.com/) with extension `ESLint`
   ```
   $ direnv allow .
   ```
-* Download [MySQL](https://dev.mysql.com/downloads/mysql/)
 
-  `MySQL` should appear at the bottom of _System Preferences_ upon successful installation
-
-  In `MySQL`, click _Initialize Database_, enter the appropriate password for "root" user and select _Use Strong Password Encryption_
-    
-  Password should correspond to the one in the `.env` file
-
-  Click _OK_ and _Start MySQL Server_
+* Spin up MySQL by running `docker-compose up`
 
 * Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server
 
   Create database by executing the following in `DBeaver`:
-  ```
+  ```sql
   CREATE DATABASE askgov;
   ```
-  Execute `$ npm run dev` to create the tables
+
+* Spin down MySQL by running `docker-compose down`
```

Comment:
This is a bit awkward. Might it be possible to have `seed.sql` to create the database, tables and seed it? Then we never have to run `docker-compose up/down` and can immediately start with `npm run dev`, followed by seeding.

Reply:
Sure; perhaps in a future PR.